README.md

```diff
@@ -1,6 +1,28 @@
 # intermezzos.github.io
 > The website for intermezzOS | https://intermezzos.github.io
 
+`instermezzos.github.io` is a static site, using [Skeleton CSS],
+served by [GitHub Pages].
+
+## Development
+
+`intermezzos.github.io` uses [`Node.JS`] and [`npm`] for development.
+
+To install development dependencies, run:
+```
```

Comment:
can you put a space between these code blocks and the paragraphs before them please?

also, since this is a static site, you don't _strictly_ need to do this, maybe just mention that it's plain old HTML and CSS and you can use what you want but this works?